app/Resources/views/base.html.twig

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>{% block title %}Profile{% endblock %}</title>
+    <title>{% block title %}OpenConext Profile{% endblock %}</title>
```

Comment:
Is the page title of every page OpenConext Profile? Should this actually be OpenConext rather than SURFconext, which is used throughout the translations?